circle.yml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  bundler:
+    without: [development]
+  post:
+    - bundle exec rake bundler:audit
```

Comment:
Don't need to run bundler audit if we are using code_climate they have built it their new stack